articles/azure-large-instances/work-with-ali-in-the-azure-portal.md

```diff
@@ -0,0 +1,271 @@
+
+```azurecli
+az baremetalinstance show --resource-group DSM05A-T550 --instance-name orcllabdsm01
+```
+
+If you're uncertain of the instance name, run the **az baremetalinstance list** command as previously described.
+
+---
+
+## Check activities of a single instance
+
+You can check the activities of a single ALI instance.
+One of the main activities recorded are restarts of the instance.
+The data listed includes:
+
+* Activity status
+* Time the activity triggered
+* Subscription ID
+* Azure user who triggered the activity
+
+ :::image type="content" source="../baremetal-infrastructure/media/connect-baremetal-infrastructure/check-activities-single-baremetal-instance.png" alt-text="Screenshot showing the BareMetal instance activities." lightbox="../baremetal-infrastructure/media/connect-baremetal-infrastructure/check-activities-single-baremetal-instance.png":::
+
+Changes to an instance's metadata in Azure also get recorded in the Activity log.
+Besides the restart, you can see the activity of **WriteBareMetalInstances**. 
+This activity makes no changes on the ALI instance itself, but documents the changes to the unit's metadata in Azure.
+
+Another activity that gets recorded is adding a tag to or deleting a tag from an instance.
+
+## Add an Azure tag to or delete an Azure tag from an instance
+
+You can add Azure tags to an ALI instance or delete them using either the Portal or Azure CLI.  
+
+### [Portal](#tab/azure-portal)
+ 
+Tags get assigned just as they do when assigning tags to VMs.
+As with VMs, the tags exist in the Azure metadata.
+Tags have the same restrictions for ALI instances as for VMs.
+ 
+Deleting tags also works the same way as for VMs. 
+Both applying and deleting a tag is listed in the ALI instance's Activity log.
+
+### [Azure CLI](#tab/azure-cli)
+
+Assigning tags to ALI instances works the same as assigning tags for VMs.
+As with VMs, the tags exist in the Azure metadata.
+Tags have the same restrictions for ALI instances as for VMs.
+
+To add tags to an ALI instance, run the [az baremetalinstance update](/cli/azure/baremetalinstance#az-baremetalinstance-update) command:
+
+```azurecli
+az baremetalinstance update --resource-group DSM05a-T550 --instance-name orcllabdsm01 --set tags.Dept=Finance tags.Status=Normal
+```
+
+Use the same command to remove a tag:
+
+```azurecli
+az baremetalinstance update --resource-group DSM05a-T550 --instance-name orcllabdsm01 --remove tags.Dept
+```
+
+---
+
+### Check properties of an instance
+
+When you acquire the instances, you can go to the Properties section to view the data collected about the instances. 
+Data collected includes:
+
+* Azure connectivity
+* Storage backend
+* ExpressRoute circuit ID
+* Unique resource ID
+* Subscription ID
+
+This information is important in support requests and when setting up a storage snapshot configuration.
+
+Image TBD in next couple of weeks - IW 
```

Comment:
@ianmsgit : Lets use the available screenshot from the word document for the time being.I dont see an update being available before GA.

Cc @akashdubey-ms

Reply:
@suzizuber : Tagging you here to address the above coment.